# Fields as the MAVlink message GLOBAL_POSITION_INT(#33)
# milliseconds
uint32 time_boot_ms
# degree *10e7	
int32 lat
int32 lon
# millimiters
int32 alt
int32 relative_alt
# m/s*100
int16 vx
int16 vy
int16 vz
# deg * 100
uint16 hdg
